test/fixtures/export/Do not caps-protect name fields #384.bibtex

```diff
@@ -179,22 +179,21 @@
 }
 
 @book{este,
-  author = {d'\relax{} Este, Beatrice},
+  author = {d'\relax Este, Beatrice},
   note = {[Este][Beatrice d']}
 }
 
 @book{medici,
-  author = {de'\mbox{} Medici, Lorenzo},
-  note = {[Medici][Lorenzo de'\mbox]}
+  author = {de' Medici, Lorenzo},
```

Comment:
Can you add it back? I'm working towards the point where I can merge your test file.